src/contributing/index.md

```diff
@@ -1,13 +1,35 @@
 # Contributing
 
-There's more to running a distribution than just writing code. This section
-explains how to be an active part of Void.
+There's more to running a distribution than just writing code.
 
-Please also visit the Void Web site for further information about [how to
-participate](https://voidlinux.org/contribute/), including our communication
-channels and how to contribute to the Void package repository.
+To contribute to the Void packages repository, start by reading the
+[CONTRIBUTING](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md)
+document in the void-packages GitHub repository.
 
-## Section Contents
+To contribute to the Void documentation, start be reading the
+[CONTRIBUTING](https://github.com/void-linux/void-docs/blob/master/CONTRIBUTING.md)
+document in the void-docs GitHub repository.
```

Comment:
Double paragraph.

Reply:
No, each para is discussing different things: the first `void-packages`, the second `void-docs`.

Reply:
Oh, I missed that. They do feel a bit repetitive, though, since they have nearly the same structure.

Also, the bottom paragraph has "be" instead of "by".

Reply:
Fair enough - have just changed the second para a bit, LMKWYT.